main.c

```diff
@@ -556,6 +556,60 @@ exiterr (GError *error)
   exit (EXIT_FAILURE);
 }
 
+/**
+ * apply_excludes:
+ * @pool: initialized pool
+ * @excludes: (out): a map of solvables
+ * @exclude_packages: names of packages to be excluded
+ * @lookasides: set of repos that will not have packages excluded
+ * @modular_pkgs: map of modular packages that exclusion does not apply to
+ *
+ * Apply excludes: this completely hides the package from any processing.
+ * Packages in lookaside are not removed, and neither are modular packages.
+ */
+static void
```

Comment:
Return type should be a map instead of one argument. Unless you have reason not to.